d2common/d2data/d2datadict/item_ratio.go

```diff
@@ -0,0 +1,77 @@
+package d2datadict
+
+import (
+	"log"
+	"strconv"
+
+	"github.com/OpenDiablo2/OpenDiablo2/d2common"
+)
+
+//ItemRatioRecord encapsulates information found in ItemRatio.txt
+//The information has been gathered from [https://d2mods.info/forum/kb/viewarticle?a=387]
+type ItemRatioRecord struct {
+	Function string
+	// 0 for classic, 1 for LoD
+	Version bool
+
+	// 0 for normal, 1 for exceptional
+	Uber          bool
+	ClassSpecific bool
+
+	// All following fields are used in item drop calculation
+	Unique           int
+	UniqueDivisor    int
+	UniqueMin        int
+	Rare             int
+	RareDivisor      int
+	RareMin          int
+	Set              int
+	SetDivisor       int
+	SetMin           int
+	Magic            int
+	MagicDivisor     int
+	MagicMin         int
+	HiQuality        int
+	HiQualityDivisor int
+	Normal           int
+	NormalDivisor    int
+}
+
+var ItemRatios map[string]*ItemRatioRecord
+
+func LoadItemRatios(file []byte) {
+	ItemRatios = make(map[string]*ItemRatioRecord)
+
+	d := d2common.LoadDataDictionary(file)
+	for d.Next() {
+		record := &ItemRatioRecord{
+			Function:         d.String("Function"),
+			Version:          d.Bool("Version"),
+			Uber:             d.Bool("Uber"),
+			ClassSpecific:    d.Bool("Class Specific"),
+			Unique:           d.Number("Unique"),
```

Comment:
can we maybe have a `ratioInfo` type? something like this:
```golang
type ratioInfo struct {
    frequency int
    divisor int
    divisorMin int
}
```
This would make the record struct simpler as we could have one for unique,  set, magic, rare, and HighQuality. for HighQuality, divisorMin could just be zero.